src/test/java/org/broadinstitute/dsde/consent/ontology/resources/TranslateResourceTest.java

```diff
@@ -8,8 +8,12 @@
 import static org.mockito.Mockito.verify;
 import static org.mockito.MockitoAnnotations.openMocks;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.api.client.http.HttpStatusCodes;
 import com.google.gson.Gson;
 import javax.ws.rs.core.Response;
+
+import org.broadinstitute.dsde.consent.ontology.cloudstore.GCSStore;
```

Comment:
Looks like these are unused, BUT, we should add some new tests to handle the new resource method, `translateParagraph`.